analysis/1/suicide_rate_distribution_analysis_byage.py

```diff
@@ -25,7 +25,7 @@
 # ========= plotting graph =========
 
 # Plotting is to visualise the suicide rate distribution in FacetGrid
-sns.set_style("dark")
+sns.set_style("ticks")
 
 p = sns.FacetGrid(suicide_rate_dataframe, hue="sex", col="age_range", col_wrap=4)
```

Comment:
How about using catplot instead of combining barplot and FacetGrid

```
sns.catplot(data = suicide_rate_dataframe, x= "sex", y = "suicide_rate", 
            hue="sex", col="age_range", col_wrap=4, kind="bar",
            order=["Male", "Female"]) 
plt.title("....")
plt.xlabel("...")
plt.ylabel("...")
```

Reply:
Thanks, Irfan.  Let me add the unit information of the plot's y-label.
The cat plot turns out with wide space in between the bars.
<img width="754" alt="Sample_ugly" src="https://user-images.githubusercontent.com/14988972/97669158-9a109180-1ad7-11eb-8bb5-a11f25297de9.png">